docs/config/templates/examples/runnableExample.template.html

```diff
@@ -18,7 +18,7 @@
     </div>
   {% endfor %}
 
-    <iframe class="runnable-example-frame" src="{$ doc.example.deployments.default.outputPath $}" name="{$ doc.example.id $}"></iframe>
+    <iframe class="runnable-example-frame" ng-src="{{getExampleIndex('{$ doc.example.deployments.default.path $}', deployment)}}" name="{$ doc.example.id $}"></iframe>
```

Comment:
OMG! Confused by Angular's `[src]`. That's embarassing 😳
The second comment is still valid (hopefully) :grin: